muikku-core-plugins/src/main/resources/META-INF/resources/scripts/src/components/evaluation/body/application/evaluation/evaluation-assessment-assignment.tsx

```diff
@@ -607,6 +608,7 @@ class EvaluationAssessmentAssignment extends React.Component<
                 selectedAssessment={this.props.selectedAssessment}
                 editorLabel={t("labels.literalEvaluation", {
                   ns: "evaluation",
+                  context: "assignment",
```

Comment:
exercise-editor.tsx file has assignment context removed but it's still present here. 

fi.json and en.json seems to be missing the `literalEvaluation_assignment` also as it has been removed by this PR